dust-lambda-extension/extensions/dust

```diff
@@ -0,0 +1,294 @@
+#!/usr/bin/env bash
+##
+## Copyright (c) 2025, Crash Override, Inc.
+##
+## This file is part of the Chalk project
+## (see https://crashoverride.com/docs/chalk)
+##
+
+set -euo pipefail
+
+# Enable debug mode only if DEBUG environment variable is set
+[[ ${DEBUG:-} == "true" ]] && set -x
+
+# Logger helpers with consistent formatting and timestamps
+log_with_level() {
+	local level="$1"
```

Comment:
tabs in shell 😭 

lets match formatting in setup-chalk-action

https://github.com/crashappsec/setup-chalk-action/blob/main/.editorconfig

shfmt honors editorconfigs so we can add that

we can copy some of the nim configs from https://github.com/crashappsec/chalk-internal/blob/main/.editorconfig to make it a global config for the repo

Reply:
fixed